src/cmds/restic/cmd_snapshots.go

```diff
@@ -87,6 +84,25 @@ func runSnapshots(opts SnapshotOptions, gopts GlobalOptions, args []string) erro
 
 	}
 
+	if gopts.JSON {
+		err := printSnapshotsJSON(list)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "error printing snapshot: %v\n", err)
+		}
+		return nil
+	}
+	printSnapshotsReadable(list)
+
+	return nil
+}
+
+//printSnapshotsReadable provides human redability
```

Comment:
Please describe what the function does, e.g.:

```
// printSnapshotsReadable prints a text table of the snapshots in list to stdout.
```

Also add a space character after the comment (`//`), and make sure that the description is a complete sentence, as described in https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences.